lib/components/gene/__tests__/about-test.js

```diff
@@ -20,7 +21,7 @@ it('shows trending artists correctly', () => {
   const gene = {
     description: 'This is some example text',
     trending_artists : [{
-      _id: '1',
+      __id: 'artist-uuid',
```

Comment:
this is what we use as a key for the components in the related_artists child array, not including it gives an error